src/components/node.js

```diff
@@ -178,7 +178,7 @@ function Node({
         shiftKey
       );
 
-      if (nodeUpdate && nodeUpdate.then) {
+      if (nodeUpdate.then) {
```

Comment:
We don't have to check if .then is defined because we know from above that it will always be defined. We now require a Promise.

Reply:
In the comment you say "We don't have to check if .then is defined because we know from above that it will always be defined", however the code checks .then (`if (nodeUpdate.then) {`) - should this be removed now?

Reply:
Good point. Thanks!